### PR TITLE
boards: qemu_arc_hs: Add separate xip & non-xip configs

### PR DESCRIPTION
**Pull request by kokas-a**
Two separate configs for qemu_arc_hs and qemu_arc_hs_xip were added.

Signed-off-by: Nikolay Agishev <agishev@synopsys.com>

### CODE REVIEW

**Comment by kokas-a** [comment]
@evgeniy-paltsev , could you please make review of this PR?

File: boards/arc/qemu_arc/qemu_arc_hs_xip.dts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2020, Synopsys, Inc. All rights reserved.
```
Comment (evgeniy-paltsev) [inline boards/arc/qemu_arc/qemu_arc_hs_xip.dts:2]: 2020 -> 2022

**Review by evgeniy-paltsev — CHANGES_REQUESTED** [review]

File: boards/arc/qemu_arc/qemu_arc_hs_xip.dts (hunk 1 above)
Comment (evgeniy-paltsev) [reply boards/arc/qemu_arc/qemu_arc_hs_xip.dts:2]: LGTM for the rest

**Comment by evgeniy-paltsev** [comment]
Duplicate of #51712

**Pull request closed without merging**